wranglerjs/index.js

```diff
@@ -93,4 +93,10 @@ compiler.run((err, stats) => {
   }
 
   writeFileSync(args["output-file"], JSON.stringify(bundle));
-});
+};
+
+if (args["watch"] === "1") {
+  compiler.watch(fullConfig.watchOptions, compilerCallback);
```

Comment:
have you tried with the wasm-pack-plugin workflow? Which has it's own watcher.

Reply:
I have not. I'm not sure how plugins that implement their own watcher interact with webpack's built in one. This may need to be revisited

Reply:
Actually I think this should be transparent, but we should try it. The code for wasm-pack-plugin is here https://github.com/wasm-tool/wasm-pack-plugin/blob/master/plugin.js.

Reply:
is this convo resolved? if so please mark it as such! or update the issue to changes requested status :)

Reply:
This was pre-webpack watcher so I believe it is resolved. Any plugin that works with webpack in watch mode will work with wrangler